.github/workflows/build-ci.yml

```diff
@@ -85,15 +94,24 @@ jobs:
         with:
           java-version: '8.0.292'
 
+      - name: Get Semver
+        id: semver
+        if: ${{ startsWith(github.ref, 'refs/heads/release/v') }}
+        run: |
+          semver=$(echo ${{ github.ref }} | tail -c +21)
+          echo "::set-output name=semver::$semver"
+
       - name: Docker meta
         id: docker_meta
         uses: docker/metadata-action@v3.3.0
         with:
           images: ghcr.io/${{ github.repository_owner }}/grpc-proxy
+          flavor: |
+            latest=false
           tags: |
             type=edge,branch=main
-            type=semver,pattern={{version}}
-            type=semver,pattern={{major}}.{{minor}}
+            type=raw,priority=950,priority=950,enable=${{ startsWith(github.ref, 'refs/heads/release/v') }},value=${{ steps.semver.outputs.semver }}
+            type=raw,enable=${{ startsWith(github.ref, 'refs/heads/release/v') }},value=latest
```

Comment:
If I'm reading this right, won't this add the tag `latest` when doing a push back to any release branch? If we have `v2.0.0`, and `v1.5.0`, and then we patch 1.5 to make `v1.5.1`, won't that get tagged `latest` when `v2.0.0` should still be `latest`?

Reply:
Yes - that's the problem w/ the `latest` tag in general - I've noticed a lot of projects fall into this trap.

https://github.com/docker/metadata-action/issues/80

It can be solved by guarding the latest tag against a specific ref - I could update this to

```
             type=raw,enable=${{ startsWith(github.ref, 'refs/heads/release/v0.2') }},value=latest 
```
, and we just need to make sure when we do version bump, we also update this.